assets/css/main.css

```diff
@@ -568,6 +568,56 @@ td.submissions-dropdown {
 .form-inline .form-group div {
     display: inline-block;
 }
+
+/* Mooc-grader questionnaire */
+
+.exercise .aplus-quiz1 .form-group {
+    margin-top: 20px;
+    margin-bottom: 20px;
+}
+
+.exercise .aplus-quiz1 .form-control {
+    margin-top: 4px;
+}
+
+.exercise .aplus-quiz1 .option-checked {
+    font-weight: bold;
+}
+
+.exercise .aplus-quiz1 .has-error .form-control {
+    border-width: 2px;
+}
+
+.exercise .aplus-quiz1 .has-error .radio,
+.exercise .aplus-quiz1 .has-error .checkbox {
+    color: #333;
+}
+
+.exercise .aplus-quiz1 .help-block {
```

Comment:
I'm concerned about the same style being applied to both the option-specific feedback and the whole-question outcome in a multiple choice (multiple correct answers) situation.

![localhost_8000_def_current_m01_introduction_07_questionnaires_ (2)](https://user-images.githubusercontent.com/6674046/87534093-3f753600-c69e-11ea-82c0-422833522607.png)

I'm wondering if this is confusing, as the bottom of the list of options includes both "Correct" and "not correct" and they're given equal prominence.

Can you think about how to distinguish them? One idea might be adding a margin and border to the left of the option-specific feedback, so that it appears indented underneath the option, but that's not the only possible solution. 

![localhost_8000_def_current_m01_introduction_07_questionnaires_ (3)](https://user-images.githubusercontent.com/6674046/87534460-b7436080-c69e-11ea-8bbb-5aa7c4bb7710.png)

Reply:
I actually tried precisely what you proposed there :D But if there was a lot of option-specific hints, it became quite messy with so many different icons. So I tried to keep them to the minimum.  

And I think you do not have the current styles from aplus running, as there the feedback for options, and about the question correctness have been at least somewhat separated :) For wrong answers the cross is red. And for correct answers the check-mark is green, and the feedback-text (eg. "Correct!") is darker, exactly for that purpose. 

Now that you mentioned this, I added the darkness for the feedback-text for wrong answers as well. So the final looks like this:

![image](https://user-images.githubusercontent.com/43036333/88024882-20622280-cb3c-11ea-87b0-83f657ff503a.png)

Reply:
I think it looks pretty nice and clear. However, is there enough separation between what is the question/answer and what is the grade or feedback. In contrast, on a pen and paper your answer is in black and the feedback/grading in red, thus creating super clear mental model what is what. At least for me, the hints look like a help text for the option, i.e, something that exists before selecting anything. Some of your drafts had a dark background for these, but that is probably too much.

In addition, is the best location for correct/wrong after the question or should it be closer to the points badge to explain that (we currently use only color for the correctness there)? It would be even possible to render the checkmark over the whole question with x20 bigger font, so it would look like it was done with a pen. Probably not a good idea though.

Is a text "your answer to this question is not correct" a bit long way to explain "incorrect"? Not a focus of the PR, so no need to address here. Just raised a thought.

Reply:
Hmm.. That is (those are) valid points to think about.

I tried those darker backgrounds, but they became very problematic with longer hints :/ Then there were just gray lines all over the place. I also tried that darkened background with just the bottom-part, but then I could not create a consistent look between checkbox-questions and other types, so I gave that up. It was also very hard to then refer to options with very long labels from that general-feedback-part. This should support longer hints and longer option-labels.  

I also would hinder from adding too much and too big extra-elements here. I think the questionnaire was quite cluttered as it is, so my instinct would be to remove extra information and not add too much extra styles there. And to try to keep it simple and suitable for different cases. Some questionnaire have very long option texts, some have very short ones, some have a lot of options, some have only few options, some have hints, some have no hints, some have only common hints, and some have only option-specific hints, some have neutral options, some have partial points, some have model-answers. and there the added complexity of the old yaml-confs that I am not able to fully test... I also tried that version were the text about correctness was next to the question, below the description or above it. But if there was an image related to the question, or the question text itself was very long, then that looked funny, and the text kind of melted into the description.

BUT, I have made these improvements now that should somehow try to answer to both of your concerns :)  

1. The crosses and checkmarks are no longer used with hints. They are there just to indicate whether the answer is correct or false, with the text "Correct!" or "Incorrect", for a consistent look and with the aim for that mental model. I think this combination of text and the clear icon, would suit for accessibility purposes as well. 

2. Hints are presented with the lighter gray color, and are in italics. This has been done to separate them from the outcome of the questionnaire. 

3. If there is a correct model answer, it is shown instead of hints. No point in showing hints if there is already the model answer showing.  

4. The classes have been checked and should reflect what is the actual content rather than the bootstrap-class. Basically anything related to feedback can now be hidden or changed if wanted, by accessing the related css-class: option-related hints, general hints and general feedback of the questionnaire. 

5. Coloring of glyphicons has been done like they were done before. So if the icon is changed to something else in the mooc-grader-end, it can be done without touching the aplus-side.

Reply:
> and are in italics

I think this was simple and very effective. At least for me hints are now clearly separate from the other things.

Thanks. I think you found a really good solutions for the concerns and the questions now look pretty clear and cool.